### ISSUE TITLE
[Components and pattern standards] Design components or patterns don't align with Design System guidelines. (04.07.2)

### ISSUE THREAD

**Issue by chaseakirby** [department-of-veterans-affairs/va.gov-team#40365]
#### VFS team name
DGIB - My Education Benefits

#### VFS product name
My Education Benefits

#### VFS feature name
My Education Benefits App

#### Point of Contact/Reviewers
Sean Sewell (@seansewell) - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components or patterns don't align with Design System guidelines.

### Issue Details
The “Continue your application” primary button is navigational so it should be a link. The “Start a new application” secondary button is also navigational so it should be a link as well. 

### Link, screenshot or steps to recreate
https://d.pr/i/L89n8v 

Guidance for action links
https://design.va.gov/experimental-design/action_links

### VA.gov Experience Standard
[Category Number 04, Issue Number 07](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

---

### Platform Recommendation
Update “Continue your application” primary button to a primary action link. Update “Start a new application” secondary button to a text link and place it below the “Continue your application” primary action link.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by cesaresantiago** [comment]
Per conversation during staging review, this change belongs in the form system. The saved application box is not controlled by MEB.

**Comment by cesaresantiago** [comment]
Closing issue as this is not controlled by MEB, but rather, the form system.
